core/src/main/java/org/infinispan/persistence/cluster/ClusterLoader.java

```diff
@@ -60,8 +61,8 @@ public MarshalledEntry load(Object key) throws PersistenceException {
       if (!isCacheReady()) return null;
 
       ClusteredGetCommand clusteredGetCommand = new ClusteredGetCommand(
-            key, cacheName, EnumUtil.EMPTY_BIT_SET, false, null,
-            cache.getCacheConfiguration().dataContainer().keyEquivalence());
+            key, cacheName, EnumUtil.bitSetOf(Flag.SKIP_OWNERSHIP_CHECK),
```

Comment:
Seems like a change that got in from the scattered cache (there I use `SKIP_OWNERSHIP_CHECK` more extensively as it has better meaning than just `CACHE_MODE_LOCAL`). It is unintentional within a scope of this PR.